x-pack/plugins/cloud_security_posture/server/saved_objects/saved_objects.ts

```diff
@@ -5,47 +5,19 @@
  * 2.0.
  */
 
-import { i18n } from '@kbn/i18n';
 import { SavedObjectsServiceSetup } from '@kbn/core/server';
-import { cspRuleSavedObjectMapping, cspRuleTemplateSavedObjectMapping } from './mappings';
-import { cspRuleMigrations, cspRuleTemplateMigrations } from './migrations';
+import { cspRuleTemplateSavedObjectMapping } from './mappings';
+import { cspRuleTemplateMigrations } from './migrations';
 import {
-  cspRuleSchemaV830,
-  cspRuleSchemaV840,
   cspRuleTemplateSchemaV830,
   cspRuleTemplateSchemaV840,
-  CspRuleTemplate,
-  CspRule,
   cspRuleTemplateSchemaV870,
+  CspRuleTemplate,
 } from '../../common/schemas';
 
-import {
-  CSP_RULE_SAVED_OBJECT_TYPE,
-  CSP_RULE_TEMPLATE_SAVED_OBJECT_TYPE,
-} from '../../common/constants';
+import { CSP_RULE_TEMPLATE_SAVED_OBJECT_TYPE } from '../../common/constants';
 
 export function setupSavedObjects(savedObjects: SavedObjectsServiceSetup) {
-  savedObjects.registerType<CspRule>({
```

Comment:
what happens once we stop registering the type to any existing csp_rule SO?

Reply:
It stops appearing in the SO objects.

I still need to check with 8.6 to 8.7 upgrade.

But 8.7 clean slate works as expected

Reply:
All checked and done 👍 

All flows work as expected (upgrading, etc)